.github/workflows/test-check-lint.yml

```diff
@@ -19,23 +19,23 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
-
+  
       - name: Run check
         run: >
           nix develop --command ./tools/code_format/check_format.py check
-
+  
   test-cpp:
     name: C++ code
     runs-on: self-hosted
     timeout-minutes: 600
     steps:
       - name: Checkout Code
         uses: actions/checkout@v3
-
+  
       - name: Run cpp tests
         run: >
-          nix develop --command bazel test //...
-
+          TEST_TMPDIR=$HOME/.bazel_test nix develop --command bazel test //...
```

Comment:
Will this line create `$HOME/.bazel_test` if the folder doesn't exist? I see in `release.yml` the line `run: mkdir -p $HOME/.bazel_test_release` before `TEST_TMPDIR=$HOME/.bazel_test_release` is used. Why is the `mkdir` called needed in `release.yml` and not in `test-check-lint.yml`?

The same question would apply to `bazel_test_opt` and `bazel_test_asan`.

Reply:
Yes. It creats that. I'll remove the mkdir command -- it was something I was trying while putting things together.